### DISCUSSION TITLE
Master Branch is not the same as the current plugin in obsidian

### DISCUSSION THREAD

**Discussion by LaMarrash** [leonhma/obsidian-functionplot#131] [General]
So I'm currently attempting to understand how to modify this code, so I can add a "Point Plot", but the current code from the Master Branch does not seem to be the same as the one you download directly from Obsidian. The code in github has some bugs, like not being able to plot a curve

**Comment by leonhma** [comment]
So... About that... I am currently working on a V2 on the branch of that same name (this next version includes many new types of plots including - yes, a point plot). The branch has many commits that aren't on the main branch, while the main branch has some that aren't on the V2 branch. As you can tell the repo is a bit all over the place currently, but I am trying to get V2 released relatively soon. There are just some minor touches that need to be done (one of them being getting the upper hand on all of those incomprehensible lines of css) and then I can release this new version. The versions are managed using the GitHub native release functionality and listed in a JSON file in the repo. If you want to test the plugin (V2) in its current form, I suggest you check out the V2 branch, although the layout may be a bit messed up.

Sorry for leaving the repo without attention for so long. I am still trying to maintain this plugin, although I am quite involved with school and other side projects at the same time.

**Reply by LaMarrash** [reply]
I really love your plug-in! Sounds amazing that updates are coming! I'll look into that branch and see if it helps me understand, because this is actually my first work with .obsidian plugins and it is actually confusing with a incomplete plug-in. Anyways, keep the good work man, this has a lot of potential!

**Reply by LaMarrash** [reply]
I just tried out the V2, and still does not plot anything. I don't know if I'm doing something wrong, but just for reference this is what I'm doing:

1. clone into the code into ../.obsidian/plugin
2. npm install on terminal (succesfully)
3. npm run build:dev on terminal (succesfully)

Not sure if there is something I must do for this plugin in particular, but it's the only feedback I can provide ATM.

**Reply by aliqut** [reply]
I've followed the exact same steps today, but using yarn and not npm, was planning on adding some features. The plot menu shows up, but the equations aren't drawn on the graph, in both the preview and the note. The plugin worked fine when I installed it through the community plugin browser.

Any updates @leonhma ?
